src/contracts/foundry-test/TestEnv.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import 'forge-std/Test.sol';
+
+import {WETH9Mock} from '@aave/periphery-v3/contracts/mocks/WETH9Mock.sol';
+import {IERC20, ERC20} from '@aave/core-v3/contracts/dependencies/openzeppelin/contracts/ERC20.sol';
+import {IStkAave} from '../facilitators/aave/stkAaveUpgrade/interfaces/IStkAave.sol';
+import {GhoAToken} from '../facilitators/aave/tokens/GhoAToken.sol';
+import {GhoToken} from '../gho/GhoToken.sol';
+import {MockedPool} from './mocks/MockedPool.sol';
+import {MockedProvider} from './mocks/MockedProvider.sol';
+import {MockedAclManager} from './mocks/MockedAclManager.sol';
+import {GhoVariableDebtToken} from '../facilitators/aave/tokens/GhoVariableDebtToken.sol';
+import {IGhoToken} from '../gho/interfaces/IGhoToken.sol';
+import {GhoDiscountRateStrategy} from '../facilitators/aave/interestStrategy/GhoDiscountRateStrategy.sol';
+import {IPool} from '@aave/core-v3/contracts/interfaces/IPool.sol';
+import {IPoolAddressesProvider} from '@aave/core-v3/contracts/interfaces/IPoolAddressesProvider.sol';
+import {IAaveIncentivesController} from '@aave/core-v3/contracts/interfaces/IAaveIncentivesController.sol';
+import {TestnetERC20} from '@aave/periphery-v3/contracts/mocks/testnet-helpers/TestnetERC20.sol';
+
+contract TestEnv is Test {
+  address faucet = 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266;
```

Comment:
where are these addresses coming from?

Reply:
The addresses are from the default mnemonic `test test ... test junk` from `anvil` Foundry node.